boom.go

```diff
@@ -24,6 +24,8 @@ import (
 	"runtime"
 	"strings"
 
+	"crypto/tls"
```

Comment:
Stdlib imports need to be grouped together.

Remove the line before crypto/tls, add a line after path/filepath.